sources/packages/web/src/views/institution/WithdrawalUpload.vue

```diff
@@ -0,0 +1,318 @@
+<template>
+  <full-page-container :full-width="true">
+    <template #header>
+      <header-navigator
+        title="Manage institution"
+        subTitle="Withdrawal Upload"
+      />
+    </template>
+    <content-group>
+      <p class="category-header-medium primary-color">
+        How to upload withdrawal files
+      </p>
+      <ul class="m-4">
+        <li>
+          Use the template provided on
+          <a
+            href="https://studentaidbc.ca/institution-officials"
+            rel="noopener"
+            target="_blank"
+            >https://studentaidbc.ca/institution-officials</a
+          >
+        </li>
+        Use the template provided on
+        https://studentaidbc.ca/institution-officials
+        <li>Please save the file as a text file ".txt"</li>
+        <li>Select the file to be uploaded</li>
+        <li>
+          Click on "Validate" to ensure your file does not have errors or
+          warnings
+        </li>
+        <li>Errors and warnings will show up below</li>
+        <li>
+          If your file has errors, please fix it in the excel file first and
+          re-upload
+        </li>
+        <li>Once there are no errors, click the "Create now" button</li>
+      </ul>
+      <p class="category-header-medium primary-color">Additional notes</p>
+      <ul class="m-4">
+        <li>
+          This will process each record as if it were a manually entered
+          withdrawal
+        </li>
+        <li>
+          Any errors will produce a list below. These errors must be fixed in
+          order to bulk upload your withdrawal file
+        </li>
+        <li>
+          When clicking "Validate" any warnings will produce a list below.
+        </li>
+      </ul>
+      <v-divider />
+      <v-form ref="uploadForm">
+        <v-row class="m-0 p-0">
+          <v-file-input
+            :loading="loading"
+            :clearable="true"
+            :accept="ACCEPTED_FILE_TYPE"
+            density="compact"
+            v-model="withdrawalFiles"
```

Comment:
![image](https://github.com/bcgov/SIMS/assets/77353155/9192e62b-7ee2-413c-a721-df7d030a0281) error can be removed by 
![image](https://github.com/bcgov/SIMS/assets/77353155/0d4d5f31-3f14-4731-8c03-5a5211602327)